config.yaml

```
@@ -253,6 +253,29 @@ Standards:
         Type: Lesson
         UID: Mo0XXv
         Path: /nested-data-and-nested-loops/nested-loops.md
+  -
+    Title: Python Debugger
+    Description: "Using the Python debugger in VS Code"
```

If the title of the topic is "Python Debugger," then the folder name describing this topic should be `python-debugger`. The other way around: if the folder name is `intermediate-python-debugging`, then this should be "Intermediate Python Debugging"

I'd like the Description to be a comma-separated list of each lesson title. In this case with the current file and folder names, here's my suggestion:



```suggestion
    Title: Intermediate Python Debugging
    Description: "Intro to Debuggers, VS Code Debugging, Debugging Pytest, Excercise??: Using Debugger"
```